test-projects/project.visivo.yml

```diff
@@ -0,0 +1,79 @@
+# V2 Spec
```

Comment:
Looking at this I think there are three types of things that we need to be consistent about:
1. Items that reference the project itself. `trace.meta.date-grain`
2. Items that reference the output of queries
3. Items that have executable code

In addition we have emerging need to combined at least 1 and 3 if we do `Selector` where you can have a matcher.